### ISSUE TITLE
Rework cu_tracing_pixel To Only Have CU-specific Functionality

### ISSUE THREAD

**Issue by alexfinnarn** [CuBoulder/express#900]
## Context

Once the tracking_pixel module is merged into the codebase, the cu_tracking_pixel code should only contain CU functionality. 

It will need to have an update hook for converting all existing tracking pixels into custom pixels. Depending on how that goes, the visibility/permissions for custom pixels might need modified in tracking_pixel.
